### DISCUSSION TITLE
Performance issue on HdfsReadFile

### DISCUSSION THREAD

**Discussion by dingqiaoyi** [facebookincubator/velox#3360] [General]
Can we reuse the Hdfs file handle here ?
https://github.com/facebookincubator/velox/blob/70fe9cf9499f8abc41c8df1afe919b3e45a73230/velox/connectors/hive/storage_adapters/hdfs/HdfsReadFile.cpp#L33-L54

![img_v2_2d7efc42-551e-4256-9414-593f61938fbg](https://user-images.githubusercontent.com/63304117/203945925-0e91a4de-7109-4461-accd-f745a80c7e7d.jpg)
![img_v2_5f787aa6-d699-4a4d-8a76-da22d59201fg](https://user-images.githubusercontent.com/63304117/203945968-7dcf087f-7618-41f6-b5a1-ee9547a0b288.jpg)

**Comment by majetideepak** [comment]
@dingqiaoyi there is some discussion on this in the PR that added HDFS support here https://github.com/facebookincubator/velox/pull/1427#issuecomment-1106714276

**Reply by dingqiaoyi** [reply]
Thanks for your help and I will look into it. Thanks!
